src/hotkey_script_palettes.lua

```diff
@@ -3,83 +3,48 @@ function plugindef()
     finaleplugin.Author = "Carl Vine"
     finaleplugin.AuthorURL = "https://carlvine.com/lua/"
     finaleplugin.Copyright = "CC0 https://creativecommons.org/publicdomain/zero/1.0/"
-    finaleplugin.Version = "v0.60.1"
+    finaleplugin.Version = "0.65"
     finaleplugin.LoadLuaOSUtils = true
-    finaleplugin.Date = "2024/01/15"
+    finaleplugin.Date = "2024/02/08"
     finaleplugin.CategoryTags = "Menu, Utilities"
     finaleplugin.MinJWLuaVersion = 0.70
     finaleplugin.Notes = [[ 
-        This is designed to help navigate the many scripts crowding your RGP Lua menu. 
+        This is designed to help navigate the many scripts crowding your _RGP Lua_ menu. 
         It provides access to Lua scripts and Finale menu items through a set of 
         easily configurable palettes (dialog windows) organised by type of activity 
         and triggered by simple "hotkey" keystrokes.
 
-        The "Hotkey Palette" principle is demonstrated expertly by Nick Mazuk at 
-        [https://www.youtube.com/@nickmazuk]. 
-        Scripts are grouped into primary categories like "Intervals", "Layers", 
-        "Notes & Chords", "Measure Items" and so on as a set of palettes triggered by keystroke. 
-        Each primary palette calls up a second palette containg scripts in related areas, 
+        The _Hotkey Palette_ principle is demonstrated expertly by 
+        Nick Mazuk [on YouTube](https://www.youtube.com/@nickmazuk). 
```

Comment:
Note that links in Markdown like this don't render in RTF as you might expect -- appears to be partly a function of how pandoc translates them and partly a function of how the native controls render the RTF (at least on Windows). As a compromise solution, I would recommend removing the brackets around the target text, so that the URL simply displays in parens.

I can send a screenshot later on of what this looks like on Windows.

Reply:
> I can send a screenshot later on of what this looks like on Windows.

Yes please. The RTF looks fine through `pandoc` and through 'show_notes_dialog' on Mac, though sometimes needs a _hard_ space to force a gap after the hyperlink.

Reply:
Well, I take it back -- your info window appears to visually render just fine, though I remember there being issues with links when I was testing the feature.

**However,** the links are not clickable, and they don't even display the URL on hover, which makes them both practically useless and uninformative.

![image](https://github.com/finale-lua/lua-scripts/assets/5248041/21125f37-4e15-40d2-8724-21d93259dffe)

(As a side note, looking at all of the text you have is making me realize that a little extra line spacing might be useful, at least on Windows. Does the text appear a bit cramped on Mac as well?)

Reply:
> the links are not clickable, and they don't even display the URL

Well that's disappointing, because they both click and hover in Mac. But worth losing to honour platform equality. Line spacing is OK on Mac - in keeping with the font size, though a tad wider wouldn't hurt. 

![RTF on Mac](https://github.com/finale-lua/lua-scripts/assets/105578039/9b395335-4575-4a85-828b-208d03e9d9da)

Reply:
@cv-on-hub Thanks for the screenshot. I'll open another thread if I can implement line spacing, so we can share how it looks on our respective platforms.

@rpatters1 Any thoughts about settings on the Windows control that would improve the hyperlink experience?